### PR TITLE
Replace start & end dates in course run list with modified date

### PR DESCRIPTION
**Pull request by dsjen**
# Original
in-dev-original

in-preview-original

published-original

**(I don't have a screenshot of the original "Studio Request" tab, but it just had a Start date)**

# New
in-dev-new

in-preview-new

published-new

studio-request-new

@edx/educator-dahlia @sstack22

### CODE REVIEW

**Comment by fysheets** [comment]
Is the course run info (eg 3T2018b) going to be added as well? Or was that not available? Or am I just not seeing it?

**Comment by dsjen** [comment]
@fysheets -- thanks for asking! It turns out that the run info isn't available in the `CourseRun` model and is calculated instead. It seemed like more effort than it was worth to add it, and I checked in with @sstack22 who agreed.

**Comment by dsjen** [comment]
I've updated the translations an the build is passing now. I also added documentation to the README on how to extract the translations and check them in for your build to pass.

File: README.rst [hunk 1]
```diff
@@ -25,6 +25,14 @@ How To Contribute
 
 Contributions are welcome. Please read `How To Contribute <https://github.com/edx/edx-platform/blob/master/CONTRIBUTING.rst>`_ for details. Even though it was written with ``edx-platform`` in mind, these guidelines should be followed for Open edX code in general.
 
+Development
+-----------
+
+Is the build failing because translations are out of date?
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Run ``make check_translations_up_to_date`` and check in the generated *.mo & *.po files to your PR.
```
Comment (fysheets) [inline README.rst:33]: 👍

**Review by fysheets — APPROVED** [review]
Looks good!

**Pull request merged**